js/iframe-coordinator/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "outDir": "./",
+    //"noImplicitAny": true,
```

Comment:
I will definitely come back and re-enable this on the second path to make sure everything is well typed.